test/powershell/Installer/WindowsInstaller.Tests.ps1

```diff
@@ -33,6 +33,6 @@ Describe "Windows Installer" -Tags "Scenario" {
             }
         }
 
-        $result | Should Not Be $null
+        $result | Should -Not -Be $null
```

Comment:
Should be `-Not -BeNullOrEmpty`

Reply:
@markekraus Can you confirm using `-BeNullOrEmpty`?